File: sample/src/main/java/com/iktwo/sample/Person.java
package com.iktwo.sample;

public class Person {
    public String firstName;
    public String lastName;
    public boolean isVI;

    public Person(String firstName, String lastName) {
        this.firstName = firstName;
        this.lastName = lastName;
    }

    public Person(String firstName, String lastName, boolean isVI) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.isVI = isVI;
    }
}

File: sample/src/main/java/com/iktwo/sample/Person.kt
package com.iktwo.sample

class Person {
    var firstName: String
    var lastName: String
    var isVI: Boolean = false

    constructor(firstName: String, lastName: String) {
        this.firstName = firstName
        this.lastName = lastName
    }

    constructor(firstName: String, lastName: String, isVI: Boolean) {
        this.firstName = firstName
        this.lastName = lastName
        this.isVI = isVI
    }
}
